contracts/libraries/SafeToL2Migration.sol

```diff
@@ -174,4 +175,22 @@ contract SafeToL2Migration is SafeStorage {
         // If the code size is greater than 0, it is a contract; otherwise, it is an EOA.
         return size > 0;
     }
+
+    /**
+     * @notice Returns a list of Safe owners.
+     * @return Array of Safe owners.
+     */
+    function getOwners() internal view returns (address[] memory) {
```

Comment:
I wonder if we need to add unit tests for this? From a FV perspective we could in theory prove that the two always return the same. May be overkill for the audit PR, but might be nice to include on `main`.

Reply:
We could add that to the PR created for `main`. Won't prefer adding it to this one as you mentioned.